src/checkUsersTry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function checkUsersTry(usersTry, numberToGuess) {
+  if (isNaN(+usersTry) || usersTry.length !== 4) {
```

Comment:
When you use some static values like `4` here especially when you use it in two or more places, it's better to move it to some variable.